documentation/source/quickstart.rst

```diff
@@ -41,6 +41,11 @@ To install the development version of cocotb:
     git clone https://github.com/cocotb/cocotb
     pip3 install -e ./cocotb
 
+.. note::
+
+    After installation, you should be able to execute ``cocotb-config``.
+    If it is not found, you need to append its location to the ``PATH`` environment variable.
+    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented :ref:`here <python:inst-alt-install-user>`_.
```

Comment:
Hmm, still not right. Either

```suggestion
    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented :ref:`here <python:inst-alt-install-user>`.
```
or perhaps
```suggestion
    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented `here <python:inst-alt-install-user>`_.
```
Maybe we should stop guessing and read the intersphinx docs.

Reply:
Should work now (works locally for me)

Reply:
Works: https://cocotb--1564.org.readthedocs.build/en/1564/quickstart.html#installation-via-pip